task-alphabet/html.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
Could you rename `html.html` -> `index.html`? 
(Most of static web servers recognize `index.html` as a root page.)